<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LightscapeWidget</class>
 <widget class="QWidget" name="LightscapeWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="mainLayout">
   <item>
    <layout class="QHBoxLayout" name="toolbarLayout">
     <item>
      <widget class="QPushButton" name="configureButton">
       <property name="text">
        <string>Configure Grid</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="toolbarSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QSplitter" name="mainSplitter">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QWidget" name="gridWidget">
      <layout class="QVBoxLayout" name="gridLayout">
       <item>
        <widget class="QTabWidget" name="layerTabs">
         <widget class="QWidget" name="topLayer">
          <attribute name="title">
           <string>Top Layer</string>
          </attribute>
          <layout class="QGridLayout" name="topGrid"/>
         </widget>
         <widget class="QWidget" name="middleLayer">
          <attribute name="title">
           <string>Middle Layer</string>
          </attribute>
          <layout class="QGridLayout" name="middleGrid"/>
         </widget>
         <widget class="QWidget" name="bottomLayer">
          <attribute name="title">
           <string>Bottom Layer</string>
          </attribute>
          <layout class="QGridLayout" name="bottomGrid"/>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="controlWidget">
      <layout class="QVBoxLayout" name="controlLayout">
       <item>
        <widget class="QGroupBox" name="deviceGroupBox">
         <property name="title">
          <string>Device Control</string>
         </property>
         <layout class="QVBoxLayout" name="deviceLayout">
          <item>
           <widget class="QComboBox" name="deviceCombo"/>
          </item>
          <item>
           <widget class="QRadioButton" name="wholeDeviceRadio">
            <property name="text">
             <string>Whole Device</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="zoneRadio">
            <property name="text">
             <string>Zone</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="ledRadio">
            <property name="text">
             <string>LED</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="selectionCombo"/>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="assignmentGroupBox">
         <property name="title">
          <string>Assignments</string>
         </property>
         <layout class="QVBoxLayout" name="assignmentLayout">
          <item>
           <widget class="QListWidget" name="assignmentList"/>
          </item>
          <item>
           <layout class="QHBoxLayout" name="buttonLayout">
            <item>
             <widget class="QPushButton" name="assignButton">
              <property name="text">
               <string>Assign</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="removeButton">
              <property name="text">
               <string>Remove</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="clearButton">
              <property name="text">
               <string>Clear</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QPushButton" name="colorButton">
            <property name="text">
             <string>Set Color</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>